### ISSUE TITLE
Ghost Prompts 

### ISSUE THREAD

**Issue by visionandamap** [trigaten/Learn_Prompting#268]
**What topic would you be interested in seeing an article/chapter about?**

**Any additional info you have about it:**

I have noticed in midjourney that sometimes a prompt element  form a previous prompt will seep into a following prompt even when it is no longer present. I call this a "ghost prompt" for lack of any other known name.

**Comment by trigaten** [comment]
ooo I havent heard of this. Could you give information on how you reproduce this? @junosha ?

**Comment by visionandamap** [comment]
Hello,

I have noticed on a few occasions that when I run long prompts and reroll
it and upscale a bunch of times,  - that there's been a few times that even
after I switch up things and remove an element from the prompt (such as for
example, "diamonds" "dragonfly wings" "vanta black")  sometimes that
element remains present in a subsequent prompt/render - even though I
removed it from the prompt. I'll get a specific example to share the next
time I realize it happening again. This has happened enough times to me
that I have taken note over the past few months.


Regards,

Peter Schuberth
*646-269-7286 mobile*

**Comment by wesmare** [comment]
I've had this Ghost prompting happen to me inside of ChatGPT. Had a fictitious business name created by the model be used multiple times on separate unrelated prompt outputs

**Comment by trigaten** [comment]
@visionandamap Could you show a specific example of this? I can't reproduce or find anything on this online.

**Comment by visionandamap** [comment]
@trigaten yeah -let me keep close attention to this and the next time i see it pop up I will gather specific details and share.

**Comment by wesmare** [comment]
@trigaten it was actually using the CharGPT API through a service that allows one to SMS text with chat GPT.  So I have to imagine it was the same thread. But I'd get the ghosting even when prompting via text hours later.

**Comment by visionandamap** [comment]
@trigaten hey - been super busy at work - but didn't forget about this. There were a few more recent examples.. I will gather up the specifics and hope to send it over the weekend.
